### PR TITLE
Use project version of markdown-link-check package

### PR DESCRIPTION
**Pull request by joaopgrassi**
## Changes

The `changelog.yaml` workflow is installing the latest version of the `markdown-link-check` package on each run, and the latest version seems to have a issue (See #1552).

Adding a make target to run this task, so then it uses the version defined in `package.json` for the repo.

## Merge requirement checklist

* [x] [CONTRIBUTING.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md) guidelines followed.
* [ ] Change log entry added, according to the guidelines in [When to add a changelog entry](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#when-to-add-a-changelog-entry).
  * If your PR does not need a change log, start the PR title with `[chore]`
* [ ] [schema-next.yaml](https://github.com/open-telemetry/semantic-conventions/blob/main/schema-next.yaml) updated with changes to existing conventions.

### CODE REVIEW

File: .github/workflows/changelog.yml [hunk 1]
```diff
@@ -76,13 +76,6 @@ jobs:
 
       # In order to validate any links in the yaml file, render the config to markdown
       - name: Render .chloggen changelog entries
-        run: make chlog-preview > changelog_preview.md
-      - name: Install markdown-link-check
-        run: npm install -g markdown-link-check
+        run: make chlog-preview 2> changelog_preview.md
```
Comment (joaopgrassi) [inline .github/workflows/changelog.yml:79]: I noticed while running this `make chlog-preview > changelog_preview.md` on Ubuntu that the file created does not contain the changelog preview, but only the actual make code.

Comment (joaopgrassi) [reply .github/workflows/changelog.yml]: Now it works :) 

image

**Comment by christophe-kamphaus-jemmic** [comment]
Is there a commit on which the changelog workflow succeeded and wasn't skipped?

**Comment by joaopgrassi** [comment]
> Is there a commit on which the changelog workflow succeeded and wasn't skipped?

No. The problem is that I added the new make target, but I can't use it in the workflow yet, since it fails because the new target is not there. So I guess we will only know when it merges. I only tested it locally and all worked.

**Review by jsuereth — APPROVED** [review]

**Review by trask — APPROVED** [review]

**Review by arminru — APPROVED** [review]

**Comment by joaopgrassi** [comment]
Let's see if it works 😅

**Pull request merged**